### PR TITLE
Changelog documentation

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*

First pass at changelog documenting currently unreleased changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -10,14 +10,21 @@ menu:
...
 ### Added
 
-- aws-iam-authenticator support
-- Allow the retention of node taints during an upgrade
+- Support for [`aws-iam-authenticator`](https://github.com/kubernetes-sigs/aws-iam-authenticator) as an authentication option in EKS-A clusters [#90](https://github.com/aws/eks-anywhere/issues/90)
+- Support for multiple worker node groups in EKS-A clusters [#840](https://github.com/aws/eks-anywhere/issues/840)
+- Support for [node taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) on EKS-A control plane and worker nodes [#189](https://github.com/aws/eks-anywhere/issues/189)
+- Support for node [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) on EKS-A control plane and worker nodes [#486](https://github.com/aws/eks-anywhere/issues/486)
+- EKS-A users can now use the IAM Role for Service Account (IRSA) feature [#601](https://github.com/aws/eks-anywhere/issues/601)
+- New command `upgrade plan cluster` lists core component changes affected by `upgrade` [#499](https://github.com/aws/eks-anywhere/issues/499)
```
Comment (g-gaston) [inline docs/content/en/docs/reference/changelog.md:18]:
```suggestion
- New command `upgrade plan cluster` lists core component changes affected by `upgrade cluster` [#499](https://github.com/aws/eks-anywhere/issues/499)
```

**Review by g-gaston — APPROVED** [review]

File: docs/content/en/docs/reference/changelog.md [hunk 2]
```diff
@@ -10,14 +10,21 @@ menu:
 
 ### Added
 
-- aws-iam-authenticator support
-- Allow the retention of node taints during an upgrade
+- Support for [`aws-iam-authenticator`](https://github.com/kubernetes-sigs/aws-iam-authenticator) as an authentication option in EKS-A clusters [#90](https://github.com/aws/eks-anywhere/issues/90)
+- Support for multiple worker node groups in EKS-A clusters [#840](https://github.com/aws/eks-anywhere/issues/840)
+- Support for [node taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) on EKS-A control plane and worker nodes [#189](https://github.com/aws/eks-anywhere/issues/189)
```
Comment (danbudris) [inline docs/content/en/docs/reference/changelog.md:15]:
```suggestion
```

**Review by jiayiwang7 — APPROVED** [review]

File: docs/content/en/docs/reference/changelog.md [hunk 3]
```diff
@@ -10,13 +10,21 @@ menu:
...
+- Support for [`aws-iam-authenticator`](https://github.com/kubernetes-sigs/aws-iam-authenticator) as an authentication option in EKS-A clusters [#90](https://github.com/aws/eks-anywhere/issues/90)
+- Support for multiple worker node groups in EKS-A clusters [#840](https://github.com/aws/eks-anywhere/issues/840)
+- Support for node [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) on EKS-A control plane and worker nodes [#486](https://github.com/aws/eks-anywhere/issues/486)
+- EKS-A users can now use the IAM Role for Service Account (IRSA) feature [#601](https://github.com/aws/eks-anywhere/issues/601)
+- New command `upgrade plan cluster` lists core component changes affected by `upgrade cluster` [#499](https://github.com/aws/eks-anywhere/issues/499)
+- Support for workload cluster's control plane and etcd upgrade through GitOps [#1007](https://github.com/aws/eks-anywhere/issues/1007)
 
 ### Changed
-
+- EKS-A now uses Cluster API (CAPI) v1.0.1, upgrading from v0.3.23.
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/changelog.md:21]:
```suggestion
- EKS-A now uses Cluster API (CAPI) v1.0.1 and v1beta1 manifests, upgrading from v0.3.23 and v1alpha3 manifests.
```

**Review by vivek-koppuru — COMMENTED** [review]
There are no other controller changes that went in too right?

File: docs/content/en/docs/reference/changelog.md [hunk 4]
```diff
@@ -10,13 +10,21 @@ menu:
 
 ### Added
 
-- aws-iam-authenticator support
-- Allow the retention of node taints during an upgrade
+- Support for [`aws-iam-authenticator`](https://github.com/kubernetes-sigs/aws-iam-authenticator) as an authentication option in EKS-A clusters [#90](https://github.com/aws/eks-anywhere/issues/90)
+- Support for multiple worker node groups in EKS-A clusters [#840](https://github.com/aws/eks-anywhere/issues/840)
+- Support for node [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) on EKS-A control plane and worker nodes [#486](https://github.com/aws/eks-anywhere/issues/486)
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/changelog.md:15]:
```suggestion
```

File: docs/content/en/docs/reference/changelog.md [hunk 5]
```diff
@@ -10,13 +10,21 @@ menu:
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/changelog.md:10]: Let's change the `Unreleased` to the new tag once that is ready

File: docs/content/en/docs/reference/changelog.md [hunk 6]
```diff
@@ -10,13 +10,21 @@ menu:
...
 
 ### Changed
-
+- EKS-A now uses Cluster API (CAPI) v1.0.1, upgrading from v0.3.23.
 - Kubernetes components and etcd now use TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 as the
   configured TLS cipher suite [#657](https://github.com/aws/eks-anywhere/pull/657), 
+  [#759](https://github.com/aws/eks-anywhere/pull/759)
+- Cilium CNI will now be upgraded by the `upgrade cluster` command [#326](https://github.com/aws/eks-anywhere/issues/326)
+- Automated git repository structure changes during Flux component `upgrade` workflow [#577](https://github.com/aws/eks-anywhere/issues/577)
+  - Upgrading a Flux managed cluster previously required manual steps. These steps have now been automated.
```
[the remark this answers is not in the document]
Comment (danbudris) [reply docs/content/en/docs/reference/changelog.md:26]: moving to add

File: docs/content/en/docs/reference/changelog.md [hunk 7]
```diff
@@ -10,13 +10,21 @@ menu:
...
+- New command `upgrade plan cluster` lists core component changes affected by `upgrade cluster` [#499](https://github.com/aws/eks-anywhere/issues/499)
+- Support for workload cluster's control plane and etcd upgrade through GitOps [#1007](https://github.com/aws/eks-anywhere/issues/1007)
 
 ### Changed
-
+- EKS-A now uses Cluster API (CAPI) v1.0.1, upgrading from v0.3.23.
 - Kubernetes components and etcd now use TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 as the
   configured TLS cipher suite [#657](https://github.com/aws/eks-anywhere/pull/657), 
+  [#759](https://github.com/aws/eks-anywhere/pull/759)
+- Cilium CNI will now be upgraded by the `upgrade cluster` command [#326](https://github.com/aws/eks-anywhere/issues/326)
```
[the remark this answers is not in the document]
Comment (danbudris) [reply docs/content/en/docs/reference/changelog.md:24]: moving to added

**Pull request merged**